scalding-core/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -603,11 +604,11 @@ object Execution {
    * but with proof that pipe matches sink
    */
 
-  sealed trait ToWrite
-  object ToWrite {
-    final case class Force[T](pipe: TypedPipe[T]) extends ToWrite
-    final case class ToIterable[T](pipe: TypedPipe[T]) extends ToWrite
-    final case class SimpleWrite[T](pipe: TypedPipe[T], sink: TypedSink[T]) extends ToWrite
+  sealed trait ToWrite extends Serializable
+  object ToWrite extends Serializable {
```

Comment:
It makes sense to have the `ToWrite` companion object be serializable if we are going to capture/serialize it, but it's not obvious to me how that would happen? Is that something we've observed?

Reply:
they were captured inside the ToIterable Execution.

Reply:
OK, makes sense.